sql/wsrep_mysqld.cc

```diff
@@ -3013,8 +3013,29 @@ bool wsrep_handle_mdl_conflict(const MDL_context *requestor_ctx,
     ticket->wsrep_report(wsrep_debug);
 
     mysql_mutex_lock(&granted_thd->LOCK_wsrep_thd);
-    if (wsrep_thd_is_toi(granted_thd) || wsrep_thd_is_in_nbo(granted_thd) ||
-        wsrep_thd_is_applying(granted_thd)) {
+    if (wsrep_thd_is_in_nbo(granted_thd) &&
+        wsrep_thd_is_applying(request_thd)) {
+      /* Here we've got the situation when NBO is in progress and applier tires
```

Comment:
Probably you meant `tries`
```suggestion
      /* Here we've got the situation when NBO is in progress and applier tries
```

Reply:
yup. done